src/marketing/variables-custom.md

```diff
@@ -21,12 +15,16 @@ _Custom Variables_
 
 1. Click <span class="btn">Add New Variable</span>.
 
-1. Enter an identifier in the **Variable Code** field, using all lowercase characters without spaces.
+1. Enter an identifier in the **Variable Code** field, using all lowercase characters without spaces. However, the underscore character can be used to represent a space. For example: `my_custom_variable`
+ 
+1. Enter a **Variable Name**, which is used for internal reference. For example: `My Custom Variable`
+   
+1. To enter the value that is associated with the variable, do one of the following:
 
-1. Enter a **Variable Name**, which is used for internal reference. Then, do one of the following:
-
-   - In the **Variable HTML Value** text field, enter any content you want to include, using basic HTML tags. This option allows you to format the value.
-   - In the **Variable Plain Value** field, enter the variable value as plain text.
+   - In the **Variable HTML Value** field, enter the variable value formatted with simple HTML tags. For example:
+   `<b>This formatted content appears in place of the variable.</b>`
```

Comment:
```suggestion
   `**This formatted content appears in place of the variable.**`
```

Reply:
Magento doesn't accept Markdown, so the example needs to show how to use HTML tags to format an attribute value.  The <b></b> tags should be visible, but the example should not be formatted, because the taqs aren't applied at this point.